data/aa_polarity/aa_polarity.py

```diff
@@ -0,0 +1,4 @@
+
+sample_sequence = "ACRZSSDP"
```

Comment:
Import appropriate packages.

Reply:
@navneeth3005 Do I need to import packages if my evaluation is a specific example like this?